extensions/go/package.json

```
@@ -50,6 +50,10 @@
         {
           "action": "externalReferences.toggle",
           "when": "panel.activeView.id == 'references'"
+        },
+        {
+          "action": "mixPreciseAndSearchBasedReferences.toggle",
```

Is the name "precise" used anywhere in the product? I think it would be better to use "semantic" for consistency with the labels in the UI
```suggestion
          "action": "mixSemanticAndSearchBasedReferences.toggle",
```

---

The only place semantic is used is in the hover, all of our docs use precise. Maybe we should wait to merge until after we conduct a user study and nail down naming.

---

Precise has scored better over semantic in user-testing. This should also be a fairly easy thing to change in the future without leaving too much junk around (this is a configuration setting controlled by a UI button and shouldn't generally be changed by hand in the user settings page).